ansible/roles/kubernetes/files/make-ca-cert.sh

```diff
@@ -94,7 +94,12 @@ octets=($(echo "${service_range}" | sed -e 's|/.*||' -e 's/\./ /g'))
 service_ip=$(echo "${octets[*]}" | sed 's/ /./g')
 
 # Determine appropriete subject alt names
-declare -a san_array=(IP:${cert_ip} IP:${service_ip} DNS:kubernetes DNS:kubernetes.default DNS:kubernetes.default.svc DNS:kubernetes.default.svc.${dns_domain} DNS:${master_name})
+declare -a san_array=(IP:${cert_ip} IP:${service_ip} DNS:kubernetes DNS:kubernetes.default DNS:kubernetes.default.svc DNS:kubernetes.default.svc.${dns_domain})
```

Comment:
Does the `cert_ip` should include all master IP addresses? or just add other master IP address in to `san_array`. If not, access to the master which is not the group[master][0] with IP, it will fail in certification.